frontend/src/components/Footer/Footer.jsx

```diff
@@ -47,14 +54,15 @@ export const Footer = (props) => {
   };
 
   //handling submit
-  const handleSubmit = (e) => {
+  const handleSubmit = async(e) => {
     e.preventDefault();
     //if isValid = true, form submission trigger
     const isValid = validation();
     if (isValid) {
-      setSubmited(true);
```

Comment:
For every new user, there should be a welcome email sent to the subscribers

Reply:
It's a frontend part... the email sent part should be done in backend.
Merge the request and i will add the email functionality in new issue.